frontend/src/Community.js

```diff
@@ -0,0 +1,56 @@
+import { Container } from "react-bootstrap";
+
+// Component: The community tab
+// route: /community
+export function Community() {
+    return (
+        <Container>
+            <h3>Community Resources</h3>
+            <h5>Code of Conduct</h5>
+            The Scivision <a href="https://github.com/alan-turing-institute/scivision/blob/main/CODE_OF_CONDUCT.md">Code of Conduct</a> sets out the expectations of people participating in the project or events.
+
+            <h5>Email Newsletter</h5>
+
+            <p>Scivision has a low-volume email newsletter, for announcements about the project and forthcoming events (one email per month or so).</p>
+
+            <p>Read previous newsletters or subscribe to the list on Substack <a href="https://scivision.substack.com">here</a>.</p>
+
+            <p>It is possible to unsubscribe at any time by following the link at the bottom of any email from the list.</p>
+
+            <h5>Slack Workspace</h5>
+
+            <p>The Scivision Slack workspace is open to anyone interested in the topics below.  Submit your email address <a href="https://docs.google.com/forms/d/e/1FAIpQLSfDPbsb_CWApnodHlNyOQMQdKhKA9meJi_SAuh8K8dVpbIiDA/viewform?usp=sf_link">here</a> (Google Forms) and one of the team will invite you.</p>
```

Comment:
I would move the how to join sentence to below the list of topics (and maybe start that sentence with 'To join, submit...')